src/Dynamic/Includes/forms/addBot.pug

```diff
@@ -0,0 +1,133 @@
+                                    option(value='' disabled) -- TypeScript --
+                                    option(value='Discordeno') disco
+                        label.label(for='short_desc') Short Description *
+                        p (Max Length: 100)
+                        .control
+                            input.input(type='text' id='short_desc' name='short_desc' required)
+                        label.label(for='long_dec_plain') Long Description Plain *
+                        p (Max Length: 5000)
+                        p This long description field will be used on bot lists that do not support Markdown or HTML in their long descriptions.
+                        .control
+                            textarea.textarea#long_desc_plain(type='text' name='long_desc_plain' required)
+                        label.label(for='long_desc_rich') Long Description Rich *
+                        p (Max Length: 5000)
+                        p This long description field will be used on bot lists that support Markdown or HTML in their long descriptions.
+                        .control
+                            textarea.textarea#long_desc_rich(type='text' name='long_desc_rich' required)
+                        label.label(for='invite_url') Bot Invite URL *
+                        .control
+                            input.input(type='link' pattern='https?://.+' id='invite_url' name='invite_url' value="https://discord.com/oauth2/authorize?client_id=" + bot.id + "&scope=bot" required)
+                        label.label(for='website_url') Website URL
+                        .control
+                            input.input(type='link' pattern='https?://.+' id='website_url' name='website_url')
+                        label.label(for='support_url') Support (Server) URL
+                        .control
+                            input.input(type='link' pattern='https?://.+' id='support_url' name='support_url')
+                        label.label(for='donate_url') Donate URL
+                        .control
+                            input.input(type='link' pattern='https?://.+' id='donate_url' name='donate_url')
+                        label.label(for='github_url') GitHub URL
+                        .control
+                            input.input(type='link' pattern='https?://.+' id='github_url' name='github_url')
+
+        .column.is-half-desktop.is-full-mobile.top
+            .card.is-box-shadow
+                .card-content
+                    .hero
+                        .hero-body.hero-slim
+                            .has-text-centered
+                                h1.title.is-size-6#error
+                                h1.title.is-size-4 Supported Lists
+                                for list in supportedLists
+                                    .columns(class='has-margin-top')
+                                        .checkbox.always(tabindex='0' role='button' class='interactive minus')
```

Comment:
Why minus? Wouldn't a 'plus' make more sense if they're toggling it to add to the specific list?